### ISSUE TITLE
[Enhancement] Show files that will be overridden

### ISSUE THREAD

**Issue by SuperSandro2000** [beetbox/beets#2852]
When importing music you could add an option to show the conflicting files paths or their names instead just the number. You are already counting the files so it should be possible to show these infos.

**Comment by sampsyo** [comment]
Hi! Can you explain what you mean by "conflicting" and "overridden"? Are you talking about duplicate albums? Maybe an example would help.

**Comment by SuperSandro2000** [comment]
I am talking about this screen
````This album is already in the library!
Old: 14 items, AAC, 125kbps, 53:12, 48.5 MiB
New: 14 items, AAC, 125kbps, 53:12, 48.5 MiB
[S]kip new, Keep both, Remove old, Merge all?
````
This is maybe a bad example as the entire album is already in my library but if only 10 of 14 files would be overwritten because I had an incomplete album I had no idea which ones are missing.
Or another example: If only 10 songs would be overwritten of maybe 16. Which ones are new?

**Comment by sampsyo** [comment]
Aha—so you’d just like to see a listing of track titles underneath the “old” line?

Sounds reasonable enough! See also https://github.com/beetbox/beets/issues/2809, which is about a more complex interface for choosing operations on duplicates.

**Comment by SuperSandro2000** [comment]
Yes. Don't need such advanced as 2809. I can imagine that this would take quite a lot of time to do.
